src/App.jsx

```diff
@@ -14,33 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const goodsFromServerWithId = goodsFromServer.map((good, index) => ({
+  name: good,
+  id: index + 1,
```

Comment:
I see, that you are doing this to add some sort of id to use with `key`. 
It is ok, but in the current example is an overkill :)